pkg/client/start.go

```diff
@@ -151,7 +151,6 @@ func (c *Client) run(newConfig bool) error {
 		return ErrNoApps
 	}
 
-	c.Config.Apps.Setup(c.Config.Timeout.Duration)
```

Comment:
This was not running during reload, and moving into setupConfig() makes sure it does. This does a bit more than add a timeout. It configures each app for http requests with their current/new config settings (after load or reload).